component/src/main/java/com/celements/structEditor/DefaultStructuredDataEditorService.java

```diff
@@ -260,4 +263,13 @@ private Object getCellValue(DocumentReference cellDocRef, XWikiDocument onDoc)
     return modelAccess.getFieldValue(cellDoc, StructuredDataEditorClass.FIELD_EDIT_FIELD_NAME);
   }
 
+  @Override
+  public List<String> getSelectTagAutocompleteJsPath() {
```

Comment:
The result is a list. Therefore please use ...Paths or ...PathList as method name